client/components/DisclaimerInfo/index.jsx

```diff
@@ -84,8 +84,9 @@ const content = {
 const DisclaimerInfo = ({ phase }) => {
     const [expanded, setExpanded] = useState(false);
 
-    const title = content[phase].title;
-    const messageContent = content[phase].messageContent;
+    const title = content[phase]?.title || content.CANDIDATE.title;
```

Comment:
Needed in the instance that a `TestPlanVersion`'s phase is marked as `DEPRECATED`, but reports did exist if it was deprecated during the candidate or recommended phase.

It could be considered as 'unapproved' in that case but should another copy be defined for this instead? Or some alert that this set of reports has been deprecated?

Reply:
My personal opinion is that different copy should be defined for `TestPlanVersion`'s with a phase of `DEPRECATED`. The copy could be similar to the `CANDIDATE` copy but could mention the deprecation, and potentially one day point to the current related `TestPlanVersion` if it exists.

Making different copy for `DEPRECATED` would also make this code easier to understand without context.

Reply:
> My personal opinion is that different copy should be defined for `TestPlanVersion`'s with a phase of `DEPRECATED`. The copy could be similar to the `CANDIDATE` copy but could mention the deprecation, and potentially one day point to the current related `TestPlanVersion` if it exists.

Yep, agreed. That would be good.

> Making different copy for `DEPRECATED` would also make this code easier to understand without context.

Thanks for your thoughts there. I agree, will draft something share.